controllers/badge.js

```diff
@@ -20,6 +21,72 @@ const getBadges = async (req, res) => {
   }
 }
 
+/**
+ * Create the Badge
+ *
+ * @param req {Object} - Express request object
+ * @param req.body {Object} - Badge object
+ * @param res {Object} - Express response object
+ */
+
+const createBadges = async (req, res) => {
+  try {
+    const getUser = await userQuery.fetchUser({ username: req.query.username })
+    const isBadgeEmpty = Object.entries(req.body).length === 0
+    const isMember = getUser.user.isMember
```

Comment:
What if I enter username as `ElonMusk`?
Line 36 would break in that case, right?
Please handle the case.